frontend/src/components/Admin/AdminShoutouts/AdminShoutouts.tsx

```diff
@@ -209,6 +224,21 @@ const AdminShoutouts: React.FC = () => {
                   className={styles.presentShoutoutMessage}
                   content={shoutout.message}
                 />
+                {(() => {
+                  let content;
```

Comment:
This looks like the exact same as the code below. It's also not the cleanest pattern to put a function and then call it here. Instead, can you abstract this code out into a separate component?